packages/testing/src/debug-tests.ts

```diff
@@ -1,13 +1,24 @@
 import { mochaCtx } from './mocha-ctx';
+const getDebug = () => {
+    const debug = (globalThis as { process?: { env: { DEBUG?: string } } })?.process?.env?.DEBUG;
+    return debug === 'true' || parseInt(debug || '0') > 0;
+};
 
-// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-const debug = (globalThis as { process?: { env: { DEBUG?: string } } })?.process?.env?.DEBUG;
-if (debug === 'true' || parseInt(debug || '0') > 0) {
+if (getDebug()) {
     // eslint-disable-next-line no-console
     console.log('Testing in debug mode');
-    (globalThis as { mocha?: Mocha })?.mocha?.timeout(0);
 }
 
+if (getDebug()) {
+    beforeEach('wrap mocha runnables to save ctx', function () {
+        this.currentTest?.timeout(0);
```

Comment:
what is this doing?

Reply:
these tests hang when in debug mode. very annoying